src/api/cpp/cvc5_kind.h

```diff
@@ -370,7 +370,7 @@ enum Kind : int32_t
    *   - 1: Size of the bit-vector that determines the semantics of the IAND
    *
    * Create with:
-   *   - `Solver::mkOp(Kind kind, uint32_t param) const`
+   *   - `Solver::mkOp(Kind kind, std::vector<uint32_t> param) const`
```

Comment:
This should be `const ... &`, and also list the initializer list overload. Or we only reference `Solver::mkOp()` here and have sphinx link to either of them.

Reply:
I'd still argue that we only list the `std::vector` variant since the `std::initializer_list` is required only to disambiguate `mkOp(kind, const std::vector<uint32_t>& args)` from `mkOp(kind, const std::string& arg)` in case of `mkOp(kind, {123})`. From a user's perspective the usage of `mkTerm` with a list of arguments does not change.

Reply:
I see. Okay.